test/Activity-test.js

```diff
@@ -141,11 +141,9 @@ describe('ActivityRepository', () => {
       ]);
   });
 
-  it('should return the positive trend dates', () => {
-    expect(activityRepository.getPositiveStepTrends()).to.deep.equal(['2019/08/20', '2019/08/23'])
+  it('should return the positive or negative trend dates', () => {
```

Comment:
I think that the `expects` should be in separate test because you are expecting separate returns here and I think @brittanystoroz said that when we are testing different outcomes, we should have a single test for each outcome. Correct me if I'm wrong, my memory is a little foggy on that.

Reply:
Yeah I bet you're right about that. I'll get to that before we merge it.

Reply:
dunzo! I separated the tests into positive/negative it blocks.